src/pyodide-py/pyodide/__init__.py

```diff
@@ -26,4 +26,6 @@
     "JsException",
     "register_js_module",
     "unregister_js_module",
+    "create_once_proxy",
+    "create_proxy",
```

Comment:
Please also add these to https://github.com/iodide-project/pyodide/blob/master/docs/usage/api-reference.md#python-api and make sure that the generated signature / documentation looks OK?

Reply:
It's a bit annoying that we have to duplicate the doc strings in order for sphinx to find them.

Reply:
I don't think it's specific to sphinx. It's also necessary to make help(create_proxy) show the correct signature.

Reply:
It isn't specific to Sphinx I agree. The issue I was referring to is the fact that our C code can't be built for native Python so any C extension docstrings currently need to be duplicated into `_core.py` so that they are present in the stubs.